lib/receptor_controller/client.rb

```diff
@@ -77,5 +77,14 @@ def receptor_log_msg(msg, account, node_id, exception = nil)
     private
 
     attr_writer :config
+
+    # Use x-rh-rbac-psk if present, x-rh-identity otherwise
+    def auth_headers
+      if config.pre_shared_key.present?
+        {'x-rh-rbac-psk' => config.pre_shared_key}
+      else
+        identity_header || {}
```

Comment:
I do like this for backwards compatibility. I'm guessing we eventually wouldn't need it at all though.  So could potentially be removed in a follow up.

Reply:
we're not sure if the PSK is applicable to development, so that's not only for backward compatibility

Reply:
To add more context, the main reason I can see leaving this in is when running locally - that way we wouldn't have to set up the receptor controller with PSK's and stuff, it would just automatically fall back to x-rh-id which the controller understands.